BUILDING.md

```diff
@@ -385,3 +385,21 @@ and [user guide](https://openssl.org/docs/fips/UserGuide-2.0.pdf).
    `/usr/local/ssl/fips-2.0`
 8. Build Node.js with `make -j`
 9. Verify with `node -p "process.versions.openssl"` (for example `1.0.2a-fips`)
+
+## Building Node.js with external core modules
+
+In general that core modules are only located in `lib/` folder.  Node can
```

Comment:
Thank you!
Some tiny fixes:
1. that core modules -> those  core modules.
2. Node -> Node.js.

Reply:
@vsemozhetbyt  Thanks a lot.   I've already update my code.